src/app/(main)/explore/page.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import RecentPostsHeader from "@components/shared/Home/RecentPostsHeader";
+import PostCard from "@components/shared/Home/PostCard";
+
+import { getPosts } from "~/lib/actions/posts";
+import { PostType } from "~/types/index";
+
+const Explore = async ({
+  searchParams,
+}: {
+  searchParams: { [key: string]: string | string[] | undefined };
+}) => {
+  const posts = await getPosts(searchParams.filter as PostType);
+  console.log(posts);
+  return (
+    <section>
+      <RecentPostsHeader />
+      {posts.map((post, index) => {
+        return <PostCard key={index} post={post} isShort={false} />;
+      })}
```

Comment:
```suggestion
      {posts.map((post) => {
        return <PostCard key={post.id} post={post} isShort={false} />;
      })}
```

Eventually when we have several filters- we'll likely start returning some results multiple times depending on the search. By using a key that is unique to the item we're mapping over, we're letting the react renderer know that this is the same item. If we just use an index, the react virtual dom won't be able to keep track of the components properly.

https://blog.anja-stricker.de/why-using-index-as-key-in-reactjs-map-function-can-cause-problems-and-how-to-fix-them

https://dev.to/shiv1998/why-not-to-use-index-as-key-in-react-lists-practical-example-3e66


It likely won't be an issue when just displaying results, but if we have the ability to use a unique identified like an ID, let's default to using it to make our code as robust as possible.